docs/source/common/lightning_module.rst

```diff
@@ -379,7 +379,7 @@ If you need to do something with all the outputs of each `validation_step`, over
 
 Validating with DataParallel
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-When training using a `accelerator` that splits data from each batch across GPUs, sometimes you might
+When training using an `accelerator` that splits data from each batch across GPUs, sometimes you might
```

Comment:
while we are at it here, can we change the word "master" to "main" just below this line, thanks.